ui/lib/open-api-explorer/addon/components/swagger-ui.js

```diff
@@ -0,0 +1,105 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import parseURL from 'core/utils/parse-url';
+import config from 'open-api-explorer/config/environment';
+import Swag from 'swagger-ui-dist';
+
+const { SwaggerUIBundle } = Swag;
+const { APP } = config;
+
+const SearchFilterPlugin = () => {
+  return {
+    fn: {
+      opsFilter: (taggedOps, phrase) => {
```

Comment:
`taggedOps` here is an immutable.js Collection so all of the manipulation below is on immutable.js data structures.